extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/lib/structure/devices/deviceParameters/impl/DeviceStateUpdateImpl.java

```diff
@@ -0,0 +1,265 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.digitalstrom.internal.lib.structure.devices.deviceParameters.impl;
+
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.structure.devices.deviceParameters.DeviceStateUpdate;
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.structure.devices.deviceParameters.constants.DeviceBinarayInputEnum;
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.structure.devices.deviceParameters.constants.SensorEnum;
+
+/**
+ * The {@link DeviceStateUpdateImpl} is the implementation of the {@link DeviceStateUpdate}.
+ *
+ * @author Michael Ochel - Initial contribution
+ * @author Matthias Siegele - Initial contribution
+ *
+ */
+public class DeviceStateUpdateImpl implements DeviceStateUpdate {
+
+    private final String UPDATE_TYPE;
+    private final Object VALUE;
+
+    /**
+     * Creates a new {@link DeviceStateUpdateImpl} with the given updateType and a value as {@link Object}.
+     *
+     * @param updateType must not be null
+     * @param value must not be null
+     */
+    public DeviceStateUpdateImpl(String updateType, Object value) {
```

Comment:
This catches all other `public DeviceStateUpdateImpl(String updateType, XXX value)` constructors, no need for them.